docs/tfengine/schemas/iam_members.md

```diff
@@ -7,3 +7,7 @@
 | Property | Description | Type | Required | Default | Pattern |
 | -------- | ----------- | ---- | -------- | ------- | ------- |
 | iam_members | [Module](https://github.com/terraform-google-modules/terraform-google-iam) | object | false | - | - |
+| iam_members.^storage_bucket\|project\|organization\|folder$ | - | array(object) | false | - | - |
```

Comment:
I suggest to put the `^storage_bucket\|project\|organization\|folder$` on the pattern column as well, so it can be used to render in templates if there's a terraform validation that needs it.

Reply:
Done. I also formatted the placeholder name differently so it's obvious that it's not the literal property name.